app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+# from time import clock_settime
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str,
+                 health : int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    @classmethod
+    def __repr__(cls) -> str:
```

Comment:
The `__repr__` method should not be a class method. It is typically an instance method used to provide a string representation of an instance. Consider renaming this method to something more appropriate, like `list_alive_animals`, and making it a regular method or a static method if it needs to operate on the class variable `alive`.